msal/src/Microsoft.Identity.Client/Internal/MsalLogger.cs

```diff
@@ -137,7 +137,7 @@ private void Log(LogLevel msalLogLevel, string messageWithPii, string messageScr
                 ? string.Empty
                 : " - " + CorrelationId;
 
-            var msalIdParameters = MsalIdHelper.GetMsalIdParameters(new PlatformInformation());
+            var msalIdParameters = MsalIdHelper.GetMsalIdParameters();
```

Comment:
>msalIdParameters [](start = 16, length = 16)

can this be "cached" for the lifetime of logger?

Reply:
yes, it can be.  backlog of notes for future refactor.